core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -176,6 +176,20 @@ class ReplicaManagerTest {
 
   @Test
   def testReadCommittedFetchLimitedAtLSO(): Unit = {
+    val props = TestUtils.createBrokerConfig(1, TestUtils.MockZkConnect)
```

Comment:
Yeah this is no longer needed in this test after a recent comment on Jun 5. I will go through the core code but typically skip the test code if they passes. Thanks for catching this!